dev-python/pypeg2/pypeg2-2.15.2.ebuild

```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-PYTHON_COMPAT=( python{2_7,3_5,3_6} )
+EAPI=7
```

Comment:
Bumping EAPI requires a revbump to the ebuild as well. So I'd suggest just updating PYTHON_COMPAT for now.

Reply:
Ok I rollback to EAPI=5

Reply:
You have it the wrong way now :P your qutebrowser is EAPI-5 while this remains EAPI-7. Also remember that there is no BDEPEND in EAPI-5, so just update PYTHON_COMPAT.

I can bump this PYTHON_COMPAT and then just merge your qutebrowser contribution if you find rebasing hard/annoying.

Reply:
Thank you @juippis I just missread you feedback